css/style.css

```diff
@@ -378,18 +384,651 @@ footer {
+
+  .project-bottom-type2 .project-lang {
+    grid-column: 1;
+    grid-row: 3;
+    display: flex;
+    width: inherit;
+    justify-content: flex-start;
+    padding-top: 1vw;
+    margin-left: 2.3vw;
+    text-align: center;
+  }
+
+  .project-bottom-type3 .project-lang {
+    grid-column: 1;
+    grid-row: 3;
+    display: flex;
+    width: inherit;
+    justify-content: flex-start;
+    padding-top: 1vw;
+    margin-left: 1.2vw;
+    text-align: center;
+  }
+
+  .project-bottom-type1 .project-lang li {
+    padding: 0.5vw 2vw;
+    font-size: 100%;
+    margin-left: 2vw;
+  }
+
+  .project-bottom-type2 .project-lang li {
+    padding: 0.5vw 2vw;
+    font-size: 100%;
+    margin-left: 2vw;
+  }
+
+  .project-bottom-type3 .project-lang li {
+    padding: 0.5vw;
+    font-size: 100%;
+    margin-left: 1vw;
+  }
+
+  .project-bottom-type1 .project-name {
+    font-size: 4.1vw;
+    padding: 0 0 0 4.16vw;
+  }
+
+  .project-bottom-type2 .project-name {
+    font-size: 2.7vw;
+    padding: 0 0 0 4.16vw;
+  }
+
+  .project-bottom-type3 .project-name {
+    font-size: 2.7vw;
+    padding: 0 0 0 2vw;
+  }
+
+  /* Projects end */
+
+  .about-title {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-bottom: 11vw;
+  }
+
+  .about-text {
+    flex: 1 1 50vw;
+  }
+
+  .about-wrapper {
+    background-image: url('../assets/img/image_dots_with_teal.svg');
+    background-position: 89% 100%;
+    background-repeat: no-repeat;
+    background-size: 15.41vw 15.41vw;
+    flex: 1 1 50vw;
+    padding: 0 0 0 11.11vw;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-content: flex-start;
+    justify-content: left;
+  }
+
+  h2 {
+    font-size: 1.38vw;
+    padding: 3.47vw 0 1.45vw 3.47vw;
+  }
+
+  .list {
+    font-size: 1.39vw;
+    letter-spacing: -0.0525em;
+    padding: 0 0 0 3.47vw;
+  }
+
+  .lang {
+    max-width: 20vw;
+    max-height: 20vw;
+    min-width: 20vw;
+    min-height: 20vw;
+    background-size: 7vw;
+    margin: 0;
+  }
+
+  .frame {
+    max-width: 20vw;
+    max-height: 20vw;
+    min-width: 20vw;
+    min-height: 20vw;
+    background-size: 7vw;
+    margin: 0;
+  }
+
+  .skill {
+    max-width: 20vw;
+    max-height: 20vw;
+    min-width: 20vw;
+    min-height: 20vw;
+    background-size: 7vw;
+    margin: 0 0 1vw;
+  }
+
+  .about-title h1 {
+    padding: 1.25vw 0 0 5vw;
+    font-size: 7.64vw;
+    font-weight: 600;
+    color: #1a2236;
+  }
+
+  .about-title .intro p {
+    padding: 2vw 11.11vw 0 5vw;
+  }
+
+  .about-title .btn {
+    padding: 0 0 0 5vw;
+  }
+
+  .btn img {
+    width: 1.7vw;
+    height: 1.7vw;
+    padding: 0 0 0 1vw;
+    vertical-align: -0.5vw;
+  }
+
+  .contact-form {
+    background-image: url('../assets/img/image-geometry_6.svg'), url('../assets/img/image-geometry_7.svg');
+    background-position: -5.3vw top, right bottom;
+    background-repeat: no-repeat, no-repeat;
+    background-size: 24.3vw 18.26vw, 24.3vw 24.3vw;
+    margin: 0;
+    width: 100vw;
```

Comment:
- [ ] Please apply what you learned from the above comment on this line here as well to fix the design overflow issue 🤗

Reply:
fixed 👍